src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculation {
+    private Color color;
+
+    public Figure(Color color) {
+        this.color = color;
+    }
+
+    public Figure() {
+
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public double getArea() {
+        return 0;
+    }
+
+    public void draw() {
+    }

```

Comment:
We actually do not need to provide implementation for abstract methods in abstract class. That is exactly the point of abstraction - we just declaring those methods, but implementation specifics are handled by subclasses. 
Reach out the [documentation](https://docs.oracle.com/javase/tutorial/java/IandI/abstract.html#:~:text=An%20abstract%20class%20is%20a,but%20they%20can%20be%20subclassed.&text=When%20an%20abstract%20class%20is,methods%20in%20its%20parent%20class.)